message/router.go

```diff
@@ -34,6 +34,11 @@ type HandlerFunc func(msg *Message) ([]*Message, error)
 // NoPublishHandlerFunc is HandlerFunc alternative, which doesn't produce any messages.
 type NoPublishHandlerFunc func(msg *Message) error
 
+// PassthroughHandler is a handler that passes the message unchanged from the subscriber to the publisher.
+var PassthroughHandler HandlerFunc = func(msg *Message) ([]*Message, error) {
```

Comment:
You could use the alias `Messages` instead of `[]*Message`, as we've done elsewhere.
If you think it'll improve readability

Reply:
`HandlerFunc` doesn't use it though, so maybe we should change them all at once?

Reply:
hmm I thought I saw some `HandlerFunc`s using `Messages` instead, maybe I was wrong